BeeSwift/GoalViewController.swift

```diff
@@ -488,7 +501,15 @@ class GoalViewController: UIViewController,  UIScrollViewDelegate, DatapointTabl
     func updateInterfaceToMatchGoal() {
         self.datapointTableController.hhmmformat = goal.hhmmFormat
         self.datapointTableController.datapoints = goal.recentData.sorted(by: {$0.updatedAt < $1.updatedAt})
+        
+        self.dueByLabel.text = goal.dueBy
+            .sorted(using: SortDescriptor(\.key))
+            .compactMap { $0.value.formatted_delta_for_beedroid }
+            .map { $0 == "✔" ? "✓" : $0 }
```

Comment:
preferring this other checkmark, which shows up well on both light and dark

It is also the same used to indicate a goal has data for the day, today (when "todayta" is true, see https://github.com/beeminder/BeeSwift/blob/745cf9875ed6b0303c46d5d8e4c0a2d44796040d/BeeSwift/GoalCollectionViewCell.swift#L27)